cmd/queue/main.go

```diff
@@ -303,7 +302,7 @@ func main() {
 	}()
 
 	// Open a websocket connection to the autoscaler
-	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.%s:%d", servingAutoscaler, system.Namespace, utils.GetClusterDomainName(), servingAutoscalerPort)
+	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.%s:%d", servingAutoscaler, "knative-serving", utils.GetClusterDomainName(), servingAutoscalerPort)
```

Comment:
Here, we must get the namespace `knative-serving` which deploy the service `autoscaler`. If we use `system.Namespace()` only can get default namespaces `knative-testing` if we set the default value in test lib. Do you have some better method to get the namespaces which `autoscaler` deployed.  @mattmoor